x/feemarket/types/codec.go

```diff
@@ -35,11 +36,13 @@ func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 	registry.RegisterImplementations(
 		(*sdk.Msg)(nil),
 		&MsgUpdateParams{},
+		&legacytypes.MsgUpdateParams{},
 	)
 	msgservice.RegisterMsgServiceDesc(registry, &_Msg_serviceDesc)
 }
 
 // RegisterLegacyAminoCodec required for EIP-712
 func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&MsgUpdateParams{}, updateParamsName, nil)
+	cdc.RegisterConcrete(&legacytypes.MsgUpdateParams{}, "ethermint/feemarket/v1/MsgUpdateParams", nil)
```

Comment:
are you sure about this naming? why it is `os/feemarket...` above?

Reply:
I think , it might have been a typo 
its should be `cosmos`

similar to 
```
	convertERC20Name = "cosmos/evm/MsgConvertERC20"
	convertCoinName  = "cosmos/evm/MsgConvertCoin" // keep it for backwards compatibility when querying txs
	updateParams     = "cosmos/evm/erc20/MsgUpdateParams"
	registerERC20    = "cosmos/evm/erc20/MsgRegisterERC20"
	toggleConversion = "cosmos/evm/erc20/MsgToggleConversion"
```

I don't think the naming is important for us, but I can change it to `ethermint/MsgUpdateParams`